src/metpy/calc/tools.py

```diff
@@ -1443,6 +1443,9 @@ def laplacian(f, axes=None, coordinates=None, deltas=None):
 
     Works for both regularly-spaced data, and grids with varying spacing.
 
+    Calculation is done using finite difference technique (centered finite difference in the middle and 
+    forward/backward at the boundaries, see metpy.calc.second_derivative for more information).
```

Comment:
Can you please wrap the lines at 95 characters and remove any trailing whitespace? That will match the style of the existing code and make the bots happy.